packages/wonder-stuff-ci/src/__tests__/mobile-release-utils.test.ts

```diff
@@ -0,0 +1,90 @@
+import {
+    compareVersions,
+    extractMobileReleaseBranch,
+} from "../mobile-release-utils";
+
+describe("#compareVersions", () => {
+    it.each(["7.8.1", "7.9.0", "7.10.0", "8.0.0"])(
+        "return 1 if version 1 is greater than version 2",
+        (testCase: any) => {
+            // Arrange
+            const version1 = testCase;
+            const version2 = "7.8.0";
+
+            // Act
+            const result = compareVersions(version1, version2);
+
+            // Assert
+            expect(result).toBe(1);
+        },
+    );
+
+    it.each(["7.7.1", "7.6.0", "6.10.0"])(
+        "return -1 if version 1 is less than version 2",
+        (testCase: any) => {
+            // Arrange
+            const version1 = testCase;
+            const version2 = "7.8.0";
+
+            // Act
+            const result = compareVersions(version1, version2);
+
+            // Assert
+            expect(result).toBe(-1);
+        },
+    );
+
+    it("return 0 if version 1 is equal to version 2", () => {
+        // Arrange
+        const version1 = "7.8.0";
+        const version2 = "7.8.0";
+
+        // Act
+        const result = compareVersions(version1, version2);
+
+        // Assert
+        expect(result).toBe(0);
+    });
+});
+
+describe("#isGetReleaseBranch", () => {
+    it.each([
+        "release/unified/7.8.0",
+        "release/android/7.8.0",
+        "release/ios/7.8.0",
+    ])("get the version from the release branch", (testCase: any) => {
```

Comment:
**question:** does TS complain if we type the `testCase` as `string` instead of `any`?

Reply:
hmm interestingly my IDE does not complain. I ran `tsc` no complaints. I even ran `yarn test` and no complaints.